### ISSUE TITLE
Need default 1st row selected in P-table and Need work with Keyboard key.

### ISSUE THREAD

**Issue by AjeetUI** [primefaces/primeng#12596]
### Describe the feature you would like to see added

1. 1st  Row should be selected by default.
2. Row selection should be work with Keyboard navigation key.

### Is your feature request related to a problem?

Yes

### Describe the solution you'd like

N/A

### Describe alternatives you have considered

N/A

### Additional context

N/A

**Comment by AjeetUI** [comment]
Any update?
